azext_iot/digitaltwins/common.py

```diff
@@ -11,6 +11,9 @@
 
 from enum import Enum
 
+MAX_ADT_CREATE_RETRIES = 5
+ADT_CREATE_RETRY_AFTER = 10
```

Comment:
Why not use the API header value `Retry-After` so the client has a dynamic adherence to a service specified wait time? If `Retry-After` is missing, then we can fallback to a hard coded wait time. The ARM [async operation spec](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers) states "3. Clients should wait for the Retry-After interval, if it was specified, or the default of **60 seconds** if it was not, before polling again."

Reply:
updated